cli/upload.py

```diff
@@ -225,7 +226,7 @@ def _wait_for_upload(procs: list) -> Optional[str]:
                 errline
                 and len(errline) > 2  # skip '* ' spinner lines
                 and errline.split("]", 1)[0].endswith(
-                    "/1 files"
+                    "1 files"
```

Comment:
This is because `gsutil` output likes slightly different when the `-m` flag isn't passed.